doc/ExtraDocuments/Conventions.md

```diff
@@ -0,0 +1,196 @@
+
+$f(u,v + t*2\pi/N_\text{FP}) = f(u,v)$ ... periodicity with period $2\pi/N_\text{FP}$ in toroidal direction
+
+The condition implies that there are only toroidal mode numbers which are mutliples of $N_\text{FP}$ (write out the series and check, if you do not believe me). This is achieved internally in the codes by using not not an toroidal angle like $\varphi_V \isin [0,2\pi[$ but rather $\zeta_V \equiv N_\text{FP}\varphi_V$ for the calculation. In the ouput of the codes this is however differently reflected
+
+- VMEC: $n$ are saved as their actual form (so one has only mode-number $n$ that are multiples of `nfp`)
+- BOOZX: same as VMEC
+- ITP Boozer: the $n$ are stored as $n_\text{actual}/$`nper` (with `nper` being the name of $N_\text{FP}$ in the ITP boozer files). They appear therefore not as multiples of the field period number but just like "-6,-5,-4,-3,-2,-1,0,1,2,3,4,5,6,...". These saved modes have to be multipled by `nper` therfore again, before evaluating the modes in a fourierseries.
+
+Note that for Tokamaks, where there is no such concept of field periods, $N_\text{FP}$ is set to $1$.
+
+## Left/right handed systems - The COCOS convention
+
+[COCOS convention](https://crppwww.epfl.ch/~sauter/cocos/) tells you if you know the cocos number, the present orientation of coordinate systems (mostly tokamaks, axissymmetric codes, ...)
+
+[Master thesis of Robert Babin](/TOPIC_MHD_Equilibria/sources/MA_Babin.pdf): see a good explanation of the nature of VMECs coordinate system (section 1.4/1.5)
+
+As seen in the coordinate system section above, besides the choice of individual coordinates, the kind of system (left/right handed) can be different as well. One has to know the present kind, as the representation (the numbers we write down, have saved in e.g. output files) of the real world quantities (like a flux) are different. In short, one should be able from knowing the system type to imagine for example in which direction the poloidal flux flows and how to encode this into the different representations. In most cases it is merely a change of sign.
+
+![Left & Right handed coordinate systems and the effect on represetation](/doc/ExtraDocuments/pictures/right_left_coordinate_systems.png)
+
+TD;LR: You have to know which convention you are in, else you can not really do anything with the numbers you are reading out of the files.
+
+### Example COCOS=2, interpreting EQDSK file content
+
+Here is an example using the provided COCOS number in an EQDSK file to reimagine the actual physical situation based on the "numbers" provided for the fluxes and safety factor. See also the quick [lookup](/doc/ExtraDocuments/Lookup_flux_coordinates.md) regarding straight field line coordinates and the safety factor (more in depth in D'heaseleer).
+
```

Comment:
**There is no COCOS number in EQDSK files!** The EQDSK “documentation” actually restricts it to odd single-digit COCOS numbers, while `field_divB0` further restricts it to COCOS 3 or 7. The last selection is up to the actual signs of the safety factor and poloidal angle.